aggregator/test/BundleServiceFees.test.ts

```diff
@@ -115,7 +116,8 @@ Fixture.test("submits bundle with sufficient token fee", async (fx) => {
     approveAndSendTokensToOrigin(fx, await wallet.Nonce(), oneToken),
   );
 
-  assertBundleSucceeds(await bundleService.add(bundle));
+  const bundleResponse: ExplicitAny = await bundleService.add(bundle);
```

Comment:
What was the use case for ExplicitAny here? Would this not be `AddBundleResponse`?

Reply:
hmm sorry, I guess hash is included in the AddBundleResponse type, Typescript is complaining for some reason when I try to access it inside the response though.